tests/requirements/py3.txt

```diff
@@ -12,7 +12,6 @@ Pillow >= 6.2.1; sys.platform != 'win32' or python_version < '3.12'
 # pylibmc/libmemcached can't be built on Windows.
 pylibmc; sys.platform != 'win32'
 pymemcache >= 3.4.0
-pytz
```

Comment:
The `backports.zoneinfo; python_version < '3.9'` on line 4 can also go now (as well as the same in setup.cfg)

Reply:
We will change it later as a part of ticket-34233.

Reply:
I don't want to mix removing deprecated features with ticket-34233.

Reply:
Fine. As you will!